<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TangYijun_guitar</string>
  </property>
  <widget class="QPushButton" name="button_C">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>93</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(146, 190, 255)</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_D">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>93</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(146, 190, 255)</string>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Em">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>93</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(146, 190, 255)</string>
   </property>
   <property name="text">
    <string>Em</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Bm7">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>93</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(146, 190, 255)</string>
   </property>
   <property name="text">
    <string>Bm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_G">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>93</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(146, 190, 255)</string>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Am">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>93</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(146, 190, 255)</string>
   </property>
   <property name="text">
    <string>Am</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
